### ISSUE TITLE
Bug: Files Issue with "Continue Where You Left Off" and Preview Pane

### ISSUE THREAD

**Issue by Lukiluc29** [files-community/Files#13065]
### Description

I have encountered a problem with Files. When I enable the "Continue where you left off" option in the settings and close Files with two tabs open, including the preview pane, upon reopening Files, nothing is displayed, and the preview pane shows "No items are selected" even though before closing, I had selected a file in one tab and a folder in another. The highlighting is no longer visible, but the path bar indicates that I am indeed in the correct folder (using the column layout).

### Steps To Reproduce

1. Enable "Continue where you left off" in Settings
2. Open two tabs for example
3. Show the preview pane in both tabs
4. Select a file or folder
5. Close Files
6. Reopen Files

### Requirements

Address any bugs or inconsistencies in the file selection and highlighting mechanism when the "Continue where you left off" option is used.

### Files Version

2.5.21.0

### Windows Version

22H2 22621.1992

### Log File

No need.

**Comment by yaira2** [comment]
This appears to be the same as #12964

**Comment by Lukiluc29** [comment]
> This appears to be the same as #12964

I don't think so because in that case, it happens when switching from "preview" to "details," whereas here, I stay on the "details" view and close and reopen the application.

**Comment by Lukiluc29** [comment]
Small update, the highlight is back but the folder is still not selected so nothing is displayed in the preview pane.
![Capture d'écran 2023-08-23 155318](https://github.com/files-community/Files/assets/128183422/e6aeb5ee-d61f-40a2-aaea-ad41dd945cca)

**Comment by yaira2** [comment]
I was able to repro this and updated the description to include the steps I followed.
